src/main/java/dan200/computercraft/shared/peripheral/monitor/ClientMonitor.java

```diff
@@ -68,13 +70,11 @@ public boolean createBuffer( MonitorRenderer renderer )
 
                 tboBuffer = OpenGlHelper.glGenBuffers();
                 OpenGlHelper.glBindBuffer( GL31.GL_TEXTURE_BUFFER, tboBuffer );
-                GL15.glBufferData( GL31.GL_TEXTURE_BUFFER, 0, GL15.GL_STATIC_DRAW );
+                monitorBuffer = null;
+
                 tboTexture = GlStateManager.generateTexture();
                 GL11.glBindTexture( GL31.GL_TEXTURE_BUFFER, tboTexture );
                 GL31.glTexBuffer( GL31.GL_TEXTURE_BUFFER, GL30.GL_R8, tboBuffer );
-                GL11.glBindTexture( GL31.GL_TEXTURE_BUFFER, 0 );
```

Comment:
Just curious as to why these lines have been removed? I assume because we end up binding them later on anyway? If so, may be worth leaving a comment here.

Reply:
It's standard in OpenGL that you bind all resources that you need when you need them, therefore we don't need to waste time unbinding everything. Though I wouldn't be surprised if Minecraft didn't follow this principle, to be honest. Seems to work fine, though.